scripts/stage_downstream_rebase.sh

```diff
@@ -0,0 +1,60 @@
+#GIT_COMMIT=c21966e7da1e95468abe04c707bba1d45bc8c56f
+#RELEASE_VERSION=1.0.0
+
+GIT_COMMIT=${GIT_COMMIT:?"Please set the GIT_COMMIT that you want to use for the rebase (from the release branch)."}
+RELEASE_VERSION=${RELEASE_VERSION:?"Please set the VERSION that you want to use for the rebase."}
+
+if ! podman login --get-login registry.redhat.io &> /dev/null; then
+  echo "Please run podman login registry.redhat.io before running this script."
+  exit 1
+fi
+
+set -e
+
+if [ -z "$OPERATOR_IMG" ]; then
+   # get the latest release from registry.redhat.io if OPERATOR_IMG isn't set
+   OPERATOR_TAG=$(skopeo list-tags docker://registry.redhat.io/rhosp-rhel8-tech-preview/osp-director-operator | jq -r '.Tags' | grep -v 'v1' | jq -r 'sort_by(.) | .[-1]')
```

Comment:
I am using skopeo version 0.1.41 and there is no list-tags command, instead simple podman works:

`  podman image search --list-tags  registry.redhat.io/rhosp-rhel8-tech-preview/osp-director-operator`

Reply:
I'm good to switch to podman if that helps. skopeo version 1.4.1 is what I have (Fedora 34)